VAMobile/src/screens/HealthScreen/Vaccines/VaccineDetails/VaccineDetailsScreen.test.tsx

```diff
@@ -112,10 +88,15 @@ context('VaccineDetailsScreen', () => {
     expect(screen.queryByText('facility 1')).toBeFalsy()
     expect(screen.queryByText('123 abc street')).toBeFalsy()
     expect(screen.queryByText('Tiburon, CA 94920')).toBeFalsy()
+  })
 
-    initializeTestInstance('HASLOCATION')
-    expect(screen.getByText('facility 1')).toBeTruthy()
-    expect(screen.getByText('123 abc street')).toBeTruthy()
-    expect(screen.getByText('Tiburon, CA 94920')).toBeTruthy()
+  it('initializes correctly for has location vaccine', async () => {
+    when(api.get as jest.Mock)
+      .calledWith(`/v0/health/locations/location1`, expect.anything())
+      .mockResolvedValue({ data: location })
```

Comment:
You have an extra `data` here so the response is like `data: { data: ...`. This can just be .mockResolvedValue({ ...location })

Reply:
thank you